modules/artifact-activation/src/test/java/org/mule/runtime/module/artifact/activation/internal/descriptor/DefaultArtifactDescriptorFactoryTestCase.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.artifact.activation.internal.descriptor;
+
+import static org.hamcrest.Matchers.hasItems;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.Test;
+import org.mule.runtime.api.deployment.meta.MuleApplicationModel;
+import org.mule.runtime.module.artifact.activation.api.descriptor.ArtifactDescriptorFactory;
+import org.mule.runtime.module.artifact.activation.api.deployable.DeployableProjectModel;
+import org.mule.runtime.module.artifact.activation.api.plugin.PluginDescriptorResolver;
+import org.mule.runtime.module.artifact.activation.api.plugin.PluginModelResolver;
+import org.mule.runtime.module.artifact.activation.internal.maven.MavenApplicationProjectModelFactory;
+import org.mule.runtime.module.artifact.api.descriptor.ApplicationDescriptor;
+import org.mule.runtime.module.artifact.api.descriptor.BundleDependency;
+import org.mule.tck.junit4.AbstractMuleTestCase;
+
+import java.io.File;
+import java.net.URISyntaxException;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+public class DefaultArtifactDescriptorFactoryTestCase extends AbstractMuleTestCase {
+
+  @Test
+  public void createApplicationDescriptor() throws Exception {
+    MavenApplicationProjectModelFactory modelFactory =
+        new MavenApplicationProjectModelFactory(getApplicationFolder("apps/shared-lib-addition-plugin-dependencies"));
```

Comment:
is `addition` instead of `additional` intentional?

Reply:
Nope, changed.